solutions/community-deploy/CORTX-Deployment.md

```diff
@@ -1,84 +1,102 @@
 # Setup K8s Cluster and Deploy CORTX Stack
 
-   Please find below steps to setup K8s cluster and deploy CORTX Stack.
+The following sections discusses how to set up the K8s cluster and deploy the CORTX Stack. The minimum prerequisites are enlisted to make sure the cluster set up process is smooth.
 
 ## Prerequisite 
-### Minimum specification required for Cortx Stack
+Following are the prerequisites to set up the K8s cluster and deploy the CORTX stack
+
+### Minimum system specifications required for Cortx Stack
    - RAM: 16GB
    - CPU: 8Core
    - DISK: Need 8 Disk with 25GB per disk
 
-### There are two types config for `solution.yaml` file manual and automated. If you have automated config type for `solution.yaml`, please make sure your vm having below drives available
+### Types of configurations available for `solution.yaml` file
+-  Manual 
+-  Automated: When using automated configuration, please make sure your VM(virtual machine) has following drives available on it:
+
 ```
-      ls /dev/sd*
+ls /dev/sd*
 ```
-   - `/dev/sdb  /dev/sdc  /dev/sdd  /dev/sde  /dev/sdf  /dev/sdg  /dev/sdh  /dev/sdi  /dev/sdj`
+`/dev/sdb  /dev/sdc  /dev/sdd  /dev/sde  /dev/sdf  /dev/sdg  /dev/sdh  /dev/sdi  /dev/sdj`
 
-### Git should be installed on the system. You can use below command for Redhat family OS to install Git
+### You must have Git installed on your system.
+You can use following command for RedHat family OS to install the Git
 
 ```
-      yum install git -y
+yum install git -y
 ```
+
 ### SELinux should be disabled
-   - Use below command to check status of SELinux.
+
+Use the following command to check status of SELinux.
 ```
-      sestatus
+sestatus
 ```
-   - If SELinux is enable then you can run below command to disabled SELinux.
+If SELinux is enable, run the following command to disable the SELinux.
 
 ```
-      sed -i 's/SELINUX=enforcing/SELINUX=disabled/' /etc/selinux/config && setenforce 0
+sed -i 's/SELINUX=enforcing/SELINUX=disabled/' /etc/selinux/config && setenforce 0
 ```
 
-   - Once above command ran successfully then reboot your system.
+Once above command runs successfully, reboot your system.
 
 ```
-      reboot
+reboot
 ```   
+
 ### Note
- 1. All nodes should be reachable over SSH with root user
+ 1. All the nodes should be reachable over SSH with root user.
  2. You also deploy CORTX on AWS EC2 instance. Please follow [CORTX Deployment on AWS](https://github.com/Seagate/cortx-re/blob/main/solutions/community-deploy/cloud/AWS/README.md)
```

Comment:
You also deploy CORTX on AWS EC2 instance
should be
You can deploy CORTX on AWS EC2 instance also.